pkg/arm/arm.go

```diff
@@ -3,6 +3,9 @@ package arm
 //go:generate go get github.com/go-bindata/go-bindata/go-bindata
 //go:generate go-bindata -nometadata -pkg $GOPACKAGE -prefix data data/...
 //go:generate gofmt -s -l -w bindata.go
+//go:generate mockgen -destination=../util/mocks/mock_arm/arm.go -package=mock_arm -source arm.go
```

Comment:
add a `//go:generate go get` or `go install` or whatever line to get mockgen above each line in each file where you call mockgen

Reply:
done